hyrax.gemspec

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
is this needed?

Reply:
Not that I know of. I'm running the long tests locally right now, so I'll amend when those complete/fail.